examples/Org.OData.Temporal.V1.objectkey-sample.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Namespace="Org.OData.Core.V1" Alias="Core" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Temporal.V1.xml">
+    <edmx:Include Alias="Temporal" Namespace="Org.OData.Temporal.V1" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Alias="this" Namespace="org.example.odata.costcenter">
+
+      <EntityType Name="CostCenter">
+        <Key>
+          <PropertyRef Name="AreaID" />
+          <PropertyRef Name="ID" />
+          <PropertyRef Name="ValidTo" />
```

Comment:
Usually it would be more common to use ValidFrom as key component since inserting new later time slices would change the ValidTo but not the ValidFrom of the existing record.

Reply:
Agreed. Here I wanted to use a "not so common" pattern that is actually used, see https://api.sap.com/api/API_COSTCENTER_SRV/resource